crates/transaction-generator-lib/src/publishing/raw_module_data.rs

```diff
@@ -490,19 +557,22 @@ pub static MODULES_FRAMEWORK_USECASES: Lazy<Vec<Vec<u8>>> = Lazy::new(|| { vec![
 pub static PACKAGE_AMBASSADOR_TOKEN_METADATA: Lazy<Vec<u8>> = Lazy::new(|| {
 	vec![
 		10, 97, 109, 98, 97, 115, 115, 97, 100, 111, 114, 1, 0, 0, 0, 0, 0, 0,
-		0, 0, 64, 70, 65, 57, 57, 67, 48, 55, 55, 55, 53, 48, 68, 48, 51, 65,
-		56, 53, 66, 55, 70, 56, 54, 65, 56, 54, 48, 49, 49, 48, 52, 70, 54, 66,
-		65, 55, 55, 48, 50, 51, 68, 70, 69, 52, 51, 49, 65, 55, 49, 57, 66, 68,
-		53, 49, 67, 57, 48, 67, 68, 50, 70, 53, 68, 70, 49, 165, 1, 31, 139, 8,
-		0, 0, 0, 0, 0, 2, 255, 141, 140, 177, 14, 130, 48, 16, 64, 247, 126, 69,
-		195, 194, 68, 193, 15, 112, 32, 70, 86, 23, 55, 194, 112, 180, 167, 65, 160, 215,
-		244, 26, 53, 49, 254, 187, 109, 26, 117, 37, 185, 229, 238, 222, 123, 189, 3, 61,
-		195, 21, 7, 97, 97, 69, 185, 151, 37, 172, 35, 48, 131, 33, 95, 138, 59, 122,
-		158, 200, 166, 243, 78, 53, 170, 41, 133, 232, 193, 24, 143, 204, 200, 131, 248, 163,
-		145, 40, 154, 231, 161, 237, 142, 69, 68, 12, 58, 180, 6, 173, 158, 18, 213, 186,
-		64, 220, 249, 152, 127, 144, 159, 35, 249, 146, 11, 105, 88, 146, 163, 84, 157, 231,
-		242, 253, 215, 144, 240, 234, 183, 23, 242, 157, 11, 103, 154, 209, 158, 198, 27, 234,
-		192, 27, 35, 33, 41, 21, 101, 39, 133, 62, 181, 7, 60, 183, 238, 0, 0, 0,
+		0, 0, 64, 68, 56, 48, 54, 49, 54, 65, 66, 70, 56, 48, 67, 49, 48, 51,
```

Comment:
Any idea why this changes?

Reply:
either compiler changes, or someone made changes without compiling? but don't exactly know :)